components/trezor_bridge/resources/app.tsx

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
```

Comment:
Lets match this file name with the output filename as well as the html file name, otherwise it can be confusing - in your BUILD.gn and in the htmll you reference this as `trezor_bridge`, but here it is called `app` and the html file is called `bridge`.

Reply:
done